README-Windows.md

```diff
@@ -0,0 +1,18 @@
+# Windows Limitations (beta notice)
+
+Brim support on Windows is currently at beta quality. There's two
+significant known limitations:
+
+1. On Windows, generation of Zeek logs from pcaps is slower than we'd like it
```

Comment:
You could remove the initial `On Windows,` phrase.

Reply:
Thanks. Done. Yeah, its previous wording was an artifact of when it was along with the other general release notes. I wanted to really drive home the point that it was a limitation that _only_ affected Windows. Now that it's broken out into a separate file with "Windows" in the name, I'm less concerned.